sycl/test/regression/msvc_crt.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clang_cl -fsycl /MD -o %t1.exe %s
+// RUN: %CPU_RUN_PLACEHOLDER %t1.exe
+// RUN: %clang_cl -fsycl /MDd -o %t2.exe %s
+// RUN: %CPU_RUN_PLACEHOLDER %t2.exe
+// RUN: %clang_cl -fsycl /MT -o %t3.exe %s
```

Comment:
```suggestion
// RUN: %clang_cl -fsycl -fsycl-targets=spir64-unknown-windows-sycldevice /MT -o %t3.exe %s
```
could be a w/a for the unrelated Windows failures.